src/dlgRoomSymbol.h

```diff
@@ -0,0 +1,66 @@
+#ifndef MUDLET_DLGROOMSYMBOL_H
+#define MUDLET_DLGROOMSYMBOL_H
+
+/***************************************************************************
+ *   Copyright (C) 2020 by Piotr Wilczynski - delwing@gmail.com            *
```

Comment:
2021 probably, and in the other file?

Reply:
Given the development period concerned it should probably be `2020-2021`...

Reply:
Is that really the case, or does only the merge time matter?